data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,743 @@
+}
+
+temple_completed = {
+   {
+      title = _("Worship Satul"),
+      body = loftomor(
+         -- TRANSLATORS: Loftomor
+         _([[May Satul warm you, Sidolus! My builders just completed the temple for Satul. Now we can start worshipping him and consecrating our priests.]]))
+   },
+   {
+      title = _("Build Temple"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm you, Loftomor. Thank you for building such a great temple. Now we need to bring food and our most precious wares to the temple to make for satisfying sacrifices. And don’t forget some golden tabards for the priests.]]))
+         .. new_objectives(obj_worship),
+   },
+}
+
+priests_consecrated = {
+   {
+      title = _("May Satul Warm Us"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[Our priests have been consecrated, and we sacrificed our finest goods for Satuls pleasure. May he warm us and show his might to our princess and these barbaric tribe.]]))
+   },
+}
+
+princess_back = {
+   {
+      title = _("Jundlina Arrives"),
+      body = jundlina(_("Jundlina"),
+         -- TRANSLATORS: Jundlina
+         _([[May Satul warm you, Sidolus. And may he forgive me my heretic behaviour. The sign of his might and the fear in the barbarians’ faces suddenly made me recognise the truth again. I was wrong about peace and coexistence, as it would have meant knightship forever. And this is not what Atlanteans are made for.]]))
+   },
+      {
+      title = _("Sidolus Relieved"),
+      body = sidolus(
+         -- TRANSLATORS: Sidolus
+         _([[May Satul warm you, Jundlina. I am glad you are here again. I hope we may trust you, as much as you will trust and faith Satul. How come the Barbarians let you go.]]))
+   },
+   {
+      title = _("Jundlina Cheers"),
+      body = jundlina(_("Jundlina"),
+         -- TRANSLATORS: Jundlina
+         _([[Sidolus, they did not let me go, I met our scout and he managed to sneak me out, as I realized, they still kept me for hostage after I expressed some concerns about our previous agreement. So we need to settle the barbarian issue now and forever on this island. There is no other way then to defeat them to live here in peace.]]))
+         .. new_objectives(obj_defeat_kalitath),
+   }
+}
+
+victory = {
+   {
+      allow_next_scenario = true,
+      title = _("Jundlina is Relieved"),
+      body = jundlina(_("Jundlina"),
+         -- TRANSLATORS: Jundlina
+         _([[Praise Satul, we have founded a new home. I deeply regret I was weak in my belief. May Satul guide me better in the future.]])
+         .. paragraphdivider() ..
+         -- TRANSLATORS: Jundlina
+         _([[Now it is time to celebrate but I expect we can’t relax long. We need to build up our society and our capability to explore the seas again.]]))
+         .. objective_text(_("Congratulations"),
+      _([[You have won this mission. Continue with the next one or keep playing for as long as you like.]]))
```

Comment:
Perhaps instead use the *de facto* standard note attached to the end of most scenarios?

Reply:
i copied the messgae from atlanteans 1. corrected it there as well